x-pack/plugins/alerting/server/authorization/alerting_authorization.ts

```diff
@@ -273,7 +277,8 @@ export class AlertingAuthorization {
 
   public async getFindAuthorizationFilter(
     authorizationEntity: AlertingAuthorizationEntity,
-    filterOpts: AlertingAuthorizationFilterOpts
+    filterOpts: AlertingAuthorizationFilterOpts,
```

Comment:
Would it make sense to fold `includeSpaceId` into the `AlertingAuthorizationFilterOpts`?

Reply:
Updated!